### PR TITLE
[sev-4403] added additional metrics @lew-gordon

### PR DESCRIPTION
**Pull request by dlasky**
Adding additional metrics

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [ ] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [ ] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: packages/destination-actions/src/destinations/engage-messaging-sendgrid/sendEmail/index.ts [hunk 1]
```diff
@@ -537,7 +539,10 @@ const action: ActionDefinition<Settings, Payload> = {
               }
             }
           }
-        })
+        }
+        statsClient?.set('actions-personas-messaging-sendgrid.request_body_size', JSON.stringify(req).length, tags)
+        const response = await request('https://api.sendgrid.com/v3/mail/send', req)
+        logger?.info(`X-Message-ID: ${response.headers.toJSON()['X-Message-ID']}`)
```
Comment (rhall-twilio) [inline packages/destination-actions/src/destinations/engage-messaging-sendgrid/sendEmail/index.ts:545]: Are there plans for rolling back this logger?

**Review by rhall-twilio — APPROVED** [review]

File: packages/destination-actions/src/destinations/engage-messaging-sendgrid/sendEmail/index.ts (hunk 1 above)
Comment (dlasky) [reply packages/destination-actions/src/destinations/engage-messaging-sendgrid/sendEmail/index.ts:545]: yeah, we just want this for the next few days, this is low traffic dest as far as i know

**Review by marinhero — APPROVED** [review]

**Comment by ahitti** [comment]
LGTM

**Comment by dlasky** [comment]
closing for https://github.com/segmentio/action-destinations/pull/1230

**Review by valerieernst — APPROVED** [review]

**Pull request closed without merging**